erpnext/assets/doctype/asset/test_asset.py

```
@@ -1828,6 +1828,70 @@ def test_multi_currency_asset_pr_creation(self):
+		company = "_Test Company with perpetual inventory"
+		set_depreciation_settings_in_company(company=company)
+		name = frappe.db.get_value(
+			"Asset Category Account",
+			filters={"parent": "Computers", "company_name": company},
+			fieldname=["name"],
+		)
+		frappe.db.set_value("Asset Category Account", name, "capital_work_in_progress_account", "")
+
+		stock_rate = 1000
+		stock_qty = 2
+		stock_amount = 2000
```

Ruff (0.14.13)

Local variable `stock_amount` is assigned to but never used

Remove assignment to unused variable `stock_amount`

(F841)